hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/protocol/DatanodeID.java

```diff
@@ -255,6 +255,13 @@ public int getXferPort() {
     return xferPort;
   }
 
+  /**
+   * @return name useful for logging or display
+   */
+  public String getDisplayName() {
```

Comment:
`DatanodeID#getXferAddrWithHostname()` is the same method, isn't it?

Reply:
> `DatanodeID#getXferAddrWithHostname()` is the same method, isn't it?

Thank you for reminding me that the two methods are indeed the same. I will make some change.